TestPages/PaddedBody/region-padding.html

```diff
@@ -6,11 +6,12 @@
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
   <link href="https://fonts.googleapis.com/css2?family=Roboto" rel="stylesheet">
   <style>
-    body, *{
-      line-height: 19px;
+    div, div *{
```

Comment:
@chaim-aharonson-applitools that's a weird rule. why not just `*`. The previous rule was also weird (`body, *` - body is targeted by *)

Reply:
the first rule was removed. But `div, div *` was not